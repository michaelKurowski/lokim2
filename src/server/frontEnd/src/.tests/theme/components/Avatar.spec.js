const React = require('react')
const Avatar = require('theme/scenes/chatpage/components/Avatar/Avatar')
const renderer = require('react-test-renderer')

describe('Avatar Component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<
            Avatar />
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
    it('Avatar icon renders correctly', () => {
        const tree = renderer.create(<
            Avatar src={`http://robohash.org/Damian`} />
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
    it('sizes Avatar renders correctly', () => {
        const tree = renderer.create(<
            Avatar height={'small'} />
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
}) 