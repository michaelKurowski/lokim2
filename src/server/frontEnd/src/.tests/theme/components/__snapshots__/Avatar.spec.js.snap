// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar Component Avatar icon renders correctly 1`] = `
<img
  src="http://robohash.org/undefined"
/>
`;

exports[`Avatar Component renders correctly 1`] = `
<img
  src="http://robohash.org/undefined"
/>
`;

exports[`Avatar Component sizes Avatar renders correctly 1`] = `
<img
  src="http://robohash.org/undefined"
/>
`;
